CHANGELOG.md

```diff
@@ -43,6 +43,7 @@
 * Add the geospatial functions `polygonsIntersectCartesian` and `polygonsIntersectSpherical` to check if two polygons intersect. [#81882](https://github.com/ClickHouse/ClickHouse/pull/81882) ([Paul Lamb](https://github.com/plamb)).
 * Support `_part_granule_offset` virtual column in MergeTree-family tables. This column indicates the zero-based index of the granule/mark each row belongs to within its data part. This addresses [#79572](https://github.com/ClickHouse/ClickHouse/issues/79572). [#82341](https://github.com/ClickHouse/ClickHouse/pull/82341) ([Amos Bird](https://github.com/amosbird)). [#82341](https://github.com/ClickHouse/ClickHouse/pull/82341) ([Amos Bird](https://github.com/amosbird))
 * Added SQL functions `colorSRGBToOkLCH` and `colorOkLCHToSRGB` for converting colours between the sRGB and OkLCH colour spaces. [#83679](https://github.com/ClickHouse/ClickHouse/pull/83679) ([Fgrtue](https://github.com/Fgrtue)).
+* Added `isValidASCII` function to check whether a string contains only valid ASCII characters. The function is optimized with SIMD instructions for better performance. [#85377](https://github.com/ClickHouse/ClickHouse/issues/85377)
```

Comment:
@rajatmohan22 could you please stop updating change-log?

Changelog is generated automatically from pull request messages for each release.

Reply:
I'm sorry, I have made the necessary changes.
I was resolving the comments here but forgot to push to the latest version in my local environment. So it was getting appended every time.
I have removed the changelog entry and made the necessary changes.